src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -133,25 +133,26 @@ private static async Task<WasmFetchResponse> CallFetch(HttpRequestMessage reques
             List<string> headerNames = new List<string>(headerCount);
             List<string> headerValues = new List<string>(headerCount);
             JSObject abortController = BrowserHttpInterop.CreateAbortController();
-            CancellationTokenRegistration? abortRegistration = cancellationToken.Register(() =>
+            CancellationTokenRegistration? abortRegistration = cancellationToken.Register(static s =>
```

Comment:
I noticed that most of the `cancellationToken.Register` usages didn't use the `state`, so I have updated them to use it.